### PR TITLE
Add SplitK support into Batched GEMM V3

### PR DESCRIPTION
**Pull request by bartekxk**

### CODE REVIEW

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdl_cshuffle_v3_multi_d.hpp [hunk 1]
```diff
@@ -639,27 +639,27 @@ struct GridwiseGemmMultiD_xdl_cshuffle_v3
...
+                b_k_split_offset = k_id * karg.KRead * karg.StrideB;
             }
             else if constexpr(is_same_v<tensor_layout::gemm::ColumnMajor, BLayout>)
             {
-                b_k_split_offset = blockIdx.z * karg.KRead;
+                b_k_split_offset = k_id * karg.KRead;
             }
 
-            if(blockIdx.z < static_cast<uint32_t>(karg.KBatch - 1))
+            if(k_id < karg.KBatch - 1)
```
Comment (geyyer) [inline include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdl_cshuffle_v3_multi_d.hpp:662]: Does < op precede subtraction? Do we need an extra parenthesis?

**Review by geyyer — COMMENTED** [review]
Looks good, just one suspicious line, could you double check?

**Review by geyyer — APPROVED** [review]

File: include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdl_cshuffle_v3_multi_d.hpp (hunk 1 above)
Comment (bartekxk) [reply include/ck/tensor_operation/gpu/grid/gridwise_gemm_xdl_cshuffle_v3_multi_d.hpp:662]: https://en.cppreference.com/w/cpp/language/operator_precedence

**Comment by zjing14** [comment]
@bartekxk  @geyyer Thanks very much

**Pull request merged**